geolocator_android/android/src/main/java/com/baseflow/geolocator/location/NmeaClient.java

```diff
@@ -42,6 +49,17 @@ public NmeaClient(@NonNull Context context, @Nullable LocationOptions locationOp
               lastNmeaMessageTime = Calendar.getInstance();
             }
           };
+
+        gnssCallback = new GnssStatus.Callback() {
+            @Override
+            public void onSatelliteStatusChanged(@NonNull GnssStatus status) {
+                gnss_satellite_count = status.getSatelliteCount();
+                gnss_satellites_used_in_fix = 0;
+                for (int i = 0; i < gnss_satellite_count; ++i)
+                    if (status.usedInFix(i))
+                        ++gnss_satellites_used_in_fix;
```

Comment:
Can you use braces here (as per [Google Java Style guide](https://google.github.io/styleguide/javaguide.html#s4.1.1-braces-always-used))?

```suggestion
                for (int i = 0; i < gnss_satellite_count; ++i) {
                    if (status.usedInFix(i)) {
                        ++gnss_satellites_used_in_fix;
                    }
                }
```